src/r_main.c

```diff
@@ -1033,9 +1033,6 @@ void R_BindRenderVariables(void)
 
   M_BindBool("translucency", &translucency, NULL, true, ss_gen, wad_yes,
              "Translucency for some things");
-  M_BindNum("tran_filter_pct", &tranmap_alpha, NULL,
```

Comment:
Is it no longer possible for users to set the translucency percentage? Is this option only for mappers?